docs/user-guide/src/bmo/install_baremetal_operator.md

```diff
@@ -0,0 +1,345 @@
+# Install Baremetal Operator
+
+Installing Baremetal Operator (BMO) involves usually three steps:
+
+1. Clone Metal3 BMO repository `https://github.com/metal3-io/baremetal-operator.git`.
+2. Adapt the configuration settings to your specific needs.
+3. Deploy BMO in the cluster with or without Ironic.
+
+*Note*: This guide assumes that a local clone of the repository is available.
+
+Configuration Settings
+======================
+
+Review and edit the file `ironic.env` found in `config/dev`.
```

Comment:
Did you mean `config/default`?

Reply:
Yes, I did. editing now

Reply:
@lentzi90, I've commited the canges, could you please review at your best convenience?

Reply:
@lentzi90 I see that markdown lint is still complaining.
I'm working on it